src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background: $base-background-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__picture {
+    display: block;
+    margin: auto;
+
+    height: 134px;
+  }
+
+  &__name {
+    margin-top: 40px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $base-text-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+
+    margin-top: 4px;
+    display: block;
+
+    color: #616070;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 16px;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price-block {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 24px;
```

Comment:
Will be better to use margins to distance elements between each other

Reply:
done